packages/headless/esbuild.mjs

```diff
@@ -128,6 +128,28 @@ const browserUmd = Object.entries(useCaseEntries).map((entry) => {
   );
 });
 
+const quanticUmd = Object.entries(useCaseEntries).map((entry) => {
```

Comment:
Since you're iteraing over each element in `useCaseEntries`, I believe you'll be generating some superfluous bundles.

I don't think you need product-recommendation, product-listing, ssr, and commerce in the context of Quantic so maybe you could skip those.